pkgs/gohugoio/hugo-extended/registry.yaml

```diff
@@ -0,0 +1,56 @@
+packages:
+  - type: github_release
+    repo_owner: gohugoio
+    repo_name: hugo
+    name: gohugoio/hugo-extended
+    asset: hugo_extended_{{trimV .Version}}_{{.OS}}-{{.Arch}}.{{.Format}}
+    description: The world’s fastest framework for building websites
+    supported_envs:
+      - linux/amd64
+      - linux/arm64
+      - windows/amd64
```

Comment:
```suggestion
      - linux
      - amd64
```

aqua itself supports only the following OS and architecture.

* windows
* linux
* darwin

--

* amd64
* arm64

So we can (should) specify envs simply.